twenty-first/src/util_types/merkle_tree.rs

```diff
@@ -674,7 +664,7 @@ pub mod merkle_tree_test {
         ) -> HashMap<MerkleTreeNodeIndex, Digest> {
             node_indices
                 .iter()
-                .map(|&i| (i, BFieldElement::new(i)))
+                .map(|&i| (i, BFieldElement::new(u64::try_from(i).unwrap())))
```

Comment:
How about the `bfe!` macro?

```suggestion
                .map(|&i| (i, bfe!(i))))
```